docs/src/alignments.md

```diff
@@ -44,38 +41,16 @@ struct AlignmentAnchor
     op::Operation
 end
 ```
-
-The next figure shows a schematic representation of an alignment object.
-
-![Alignment representation](assets/alignment.svg)
+The start of the sequence range to which the given operation is applied is determined
+by the sequence coordinates of the previous anchor in the series.
 
 Every alignment starts with a special `OP_START` operation which is used to give
 the position in the reference and query prior to the start of the alignment, or
 0, if the alignment starts at position 1.
 
-For example, consider the following alignment:
+For example, consider the following alignment and its representation as a series of anchors:
 
-                  0   4        9  12 15     19
-                  |   |        |  |  |      |
-        query:     TGGC----ATCATTTAACG---CAAG
-    alignment:     ....----.....---...---....
-    reference: AGGGTGGCATTTATCAG---ACGTTTCGAGAC
-                  |   |   |    |     |  |   |
-                  4   8   12   17    20 23  27
-
-Using anchors we would represent this as the following series of anchors:
-```julia
-[
-    AlignmentAnchor( 0,  4,  0, OP_START),
-    AlignmentAnchor( 4,  8,  4, OP_MATCH),
-    AlignmentAnchor( 4, 12,  8, OP_DELETE),
-    AlignmentAnchor( 9, 17, 13, OP_MATCH),
-    AlignmentAnchor(12, 17, 16, OP_INSERT),
-    AlignmentAnchor(15, 20, 19, OP_MATCH),
-    AlignmentAnchor(15, 23, 22, OP_DELETE),
-    AlignmentAnchor(19, 27, 26, OP_MATCH),
-]
-```
+![Alignment representation](assets/alignment.svg)
```

Comment:
> * I think it is important for the dashes and dots in the alignment row to align with the query and reference.

I think it was the font problem. Now I have switched SVG to the embedded fonts, so the alignment should be ok. Let me know if some alignment issues still remain.

> * I'm unconvinced that the enumeration in the circles adds information. So instead of enumerating the operations/anchors in the alignment row, could the aln position be shown instead?

The numbers refer to the exact row in the table. If replaced by the alignment position (also then the alignment position should be moved to the right side of the block), matching operations on the diagram and in the table becomes less straightforward. So I would rather keep it as it is.